cloud/openstack/os_keystone_endpoint.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/python
+
+# Copyright (c) 2016 Helicom
```

Comment:
Can we get a contact information on this, in case of future relicensing requests?